tensorflow/python/keras/benchmarks/keras_examples_benchmarks/mnist_hierarchical_rnn_benchmark_test.py

```diff
@@ -117,16 +112,21 @@ def benchmark_hrnn_mnist_bs_512(self):
     self.report_benchmark(
         iters=run_iters, wall_time=wall_time, metrics=metrics, extras=extras)
 
-  def benchmark_hrnn_mnist_bs_1024(self):
-    """Measure performance with batch_size=1024 and run_iters=3."""
+  def benchmark_hrnn_mnist_bs_1024_gpu_2(self):
+    """Measure performance with batch_size=1024, run_iters=4, gpu=2 and
+
+    distribution_strategy='mirrored'
+    """
     batch_size = 1024
-    run_iters = 3
+    run_iters = 4
```

Comment:
Ditto as above: let's keep the `run_iters` the same for the same batch_size and the same model.